### DISCUSSION TITLE
新人自我介绍

### DISCUSSION THREAD

**Discussion by Serendipity96** [alc-beijing/SIG#10] [通用论坛]
大家好，我是Serendipity96，是开源项目 [LogicFlow](https://github.com/didi/LogicFlow) 的运营同学，也是 LogicFlow 的 contributor。我对开源社区一直很感兴趣，也经常关注国内的开源项目，喜欢开源社区的文化和协作方式。很高兴能成为 ALC 北京的志愿者，希望通过 ALC 我对开源有更深的了解，也希望在社区中能够发挥自己所长，让更多人了解开源，参与到开源中来。谢谢~

**Comment by JesseZhou-1** [comment]
欢迎欢迎！😊

**Reply by WillemJiang** [reply]
我们很缺运营的小伙伴，欢迎分享你的运营心得。
